src/logo-system/brand-hierarchy/index.md

```diff
@@ -29,7 +29,7 @@ In both horizontal and stacked lock-ups, the space between the wordmark and prod
 {% grid { columns: { mobile: 2, desktop: 2 } } %}
 
 {% gridCell { verticalAlign: "end" } %}
-![Various diagrams of the GOV.UK Pay lock-up in black showing how the dot is used to set spacing between the product name and wordmark. Horizontal and stacked lock-ups are shown. The dot between 'GOV' and 'UK' is Primary blue.](./lockup-1.svg)
+![Various diagrams of the GOV.UK Pay lock-up showing how the dot is used to set spacing between the product name and wordmark. It is one dot between them when horizontally as well as vertically stacked. The shape of the G from the wordmark is showing spacing around the lock-up.](./lockup-1.svg)
```

Comment:
This is the one text I changed the most, for these reasons:

* I thought the "G" shapes were important as they were not referenced anywhere else.
  * Ideally this should also be somewhere in the body copy.
* I also opted to describe more what the dot does, although it is sort of described more in the text below.
* I removed the references to the colours mostly to shorten the text as my additions made it longer. The colours are less important here, so I think that is okay.

Reply:
Although, it's 339 characters, which is quite a lot. I guess that's another indication that this should ideally be in the body copy.
One way of shortening it at least a tiny bit (to 315 characters): Instead of "are used to show" we can just say "show".
We could also shorten it by removing the dot spacing altogether, as that is described in the text above the image (although not as well as here). It's specifically the "G" spacing that isn't mentioned anywhere.